Gemfile

```diff
@@ -13,13 +13,6 @@ def location_for(place, fake_version = nil)
 end
 
 
-# We don't put beaker in as a test dependency because we
-# don't want to create a transitive dependency
-group :acceptance_testing do
-  gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '>= 4.30')
```

Comment:
This is the only use of `location_for`. If you drop it here, you might as well drop it from `Gemfile` altogether